samples/apps/forum/src/ccf/util.ts

```diff
@@ -1,94 +1,13 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Probably `util` is not the best name. Since these are all KV helpers, maybe something with kv in the name?

Reply:
app.ts to match the C++ ccfapp namespace?

Reply:
I don't think that's a good name. We may have more modules in the future and collectively they may be in the "ccfapp" namespace but modules != namespace.

Reply:
Let's ignore this for now and think about it again when we make it into a library.